pkg/commands/backend/describe.go

```diff
@@ -127,6 +127,15 @@ func (c *DescribeCommand) print(out io.Writer, b *fastly.Backend) error {
 	fmt.Fprintf(out, "Min TLS version: %v\n", fastly.ToValue(b.MinTLSVersion))
 	fmt.Fprintf(out, "Max TLS version: %v\n", fastly.ToValue(b.MaxTLSVersion))
 	fmt.Fprintf(out, "SSL ciphers: %v\n", fastly.ToValue(b.SSLCiphers))
+	fmt.Fprintf(out, "HTTP KeepAlive Timeout: %v\n", fastly.ToValue(b.KeepAliveTime))
+	if b.TCPKeepAliveEnable == nil {
+		fmt.Fprintf(out, "TCP KeepAlive Enabled:\n")
+	} else {
+		fmt.Fprintf(out, "TCP KeepAlive Enabled: %v\n", fastly.ToValue(b.TCPKeepAliveEnable))
+	}
```

Comment:
I think this is likely to be confusing for users, as "TCP KeepAlive Enabled:" with no value is going to be interpreted as "enabled".

If it is necessary for the field to always appear even if the value is `nil`, then the label should be "TCP KeepAlive" and the value should be "unset" or something like that. If it's OK for the field to not appear at all if the value is `nil`, then that would be preferable.

Whichever change is made, the same change should be made in `pkg/text/backend.go`.

Reply:
I've made it `unset` in b899fb9bc41cec92c81cdf66609ddb4627d8b8a9 so that it's always present in case someone greps for it or is scanning output in a support ticket.